docs/_ug/Layout.md

```diff
@@ -1,15 +1,15 @@
 <!-- markdownlint-disable-file first-line-h1 -->
 
 <!-- TODO: Update UI Images after UI updates are implemented -->
-Let us look at the layout of the different components of FoodRem.
+When you launch FoodRem, FoodRem appears on your screen as a [Graphical User Interface](#graphical-user-interface), or GUI. Let's look at the layout of the different components of FoodRem.
```

Comment:
Is this fragment supposed to link to the glossary?

Reply:
Yes!